sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/samples/Sample3_CreateAndDeploy.md

```diff
@@ -0,0 +1,184 @@
+# Create and Deploy Question Answering Projects
+
+This sample demonstrates how to create and deploy Question Answering projects. To get started, you'll need to create a Question Answering service endpoint and an API key. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md) for links and instructions.
+
+To create, deploy, or perform any other authoring actions for Question Answering projects, you need to first create a `QuestionAnsweringProjectsClient` using an endpoint and API key. These can be stored in an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:QuestionAnsweringProjectsClient_Create
+Uri endpoint = new Uri("https://myaccount.api.cognitive.microsoft.com");
```

Comment:
As mentioned above (won't repeat, so please check all instances) let's use an obvious placeholder like you do below for the `{api-key}`.